_layouts/default.html

```diff
@@ -1,17 +1,12 @@
 <!doctype html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>{{ page.title }}</title>
-    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="/assets/css/styles.css">
-  </head>
+  {% include head.html %}
   <body>
-    {% include header.html %}
+  	{% if page.title == 'Home' %}
```

Comment:
Fix the indentation here

Reply:
sorry, I didn't get that... should I remove the indentation ?, I used two spaces

Reply:
You added 2 extra spaces here